tests/fixtures/optimizer/qualify_columns_create.sql

```diff
@@ -0,0 +1,29 @@
+# title: Qualify columns for a create with CTE
+# execute: false
+WITH cte AS (SELECT a FROM db.y) CREATE TABLE s AS SELECT * FROM cte;
+WITH cte AS (SELECT a AS a FROM db.y AS y) CREATE TABLE s AS SELECT cte.a AS a FROM cte;
```

Comment:
Why doesn't `cte` have an alias in the final `SELECT` here? Seems a bit off. Ditto for subsequent tests.